src/components/panel/panel.spec.js

```diff
@@ -1222,6 +1287,45 @@ describe('$mdPanel', function() {
           panel2.close();
           panel3.close();
         });
+
+    it('should close the first open panel of any group that the panel is in ' +
+        'when more than the maxium number of panels is opened', function() {
+          $mdPanel.newPanelGroup('test1', {
+            maxOpen: 2
+          });
+          $mdPanel.newPanelGroup('test2', {
+            maxOpen: 2
```

Comment:
Please choose different `maxOpen` for each group so we can also verify the maxes are being set of the right panels.